src/schema/v1/me/followed_fairs.ts

```diff
@@ -1,23 +1,14 @@
 import Fair from "schema/v1/fair"
-import { IDFields } from "schema/v1/object_identification"
 
 import { pageable, getPagingParameters } from "relay-cursor-paging"
 import { connectionDefinitions, connectionFromArraySlice } from "graphql-relay"
-import { GraphQLObjectType, GraphQLFieldConfig } from "graphql"
+import { GraphQLFieldConfig } from "graphql"
 import { ResolverContext } from "types/graphql"
 
-const FollowedFairEdge = new GraphQLObjectType<any, ResolverContext>({
-  name: "FollowedProfileEdge",
-  fields: {
-    ...IDFields,
-  },
-})
-
 export const FollowedFairConnection = connectionDefinitions({
   name: "FollowedFair",
-  // FIXME: 'edgeType' does not exist in type 'ConnectionConfig'
-  // @ts-ignore
-  edgeType: FollowedFairEdge,
+  // This never ended up being used in the underlying lib.
+  // edgeType: FollowedFairEdge,
```

Comment:
@zephraph I should’ve called this out, but these v1 changes are no-ops. This never actually worked and enabling it breaks things.